module.go

```diff
@@ -87,7 +87,7 @@ func (fs *FS) Provision(ctx caddy.Context) error {
 		if fs.Endpoint != "" {
 			o.BaseEndpoint = aws.String(fs.Endpoint)
 		}
-
+		o.DisableLogOutputChecksumValidationSkipped = true
```

Comment:
Would you mind adding that link here as a reference?

THanks!

Reply:
done